terraform/10-sendgrid.tf

```diff
@@ -128,6 +128,24 @@ resource "azurerm_key_vault" "keyvault" {
     ]
   }
 
+    access_policy {
```

Comment:
formatting looks broken, can you run a terraform fmt?

Reply:
done